src/DependencyInjection/ControllerAwareTrait.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/*
+ * This file is part of Symplify
+ * Copyright (c) 2016 Tomas Votruba (http://tomasvotruba.cz).
+ */
+
+namespace Symplify\ControllerAutowire\DependencyInjection;
+
+trait ControllerAwareTrait
```

Comment:
So you wouldn't create ControllerAwareTrait at all? I suppose, that if you want use everything as before, just with proper injectation, you just can put this trait into your controller. You would rather use every trait separately?